src/vs/workbench/contrib/terminal/browser/terminalView.ts

```diff
@@ -129,7 +129,8 @@ export class TerminalViewPane extends ViewPane {
 		}
 	}
 
-	override renderBody(container: HTMLElement): void {
+	// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Wondering why only this line has to be disabled and not others

Reply:
Yeah good point, was wondering the same @Tyriar

Reply:
Because terminal has an eslintrc file scopes to the folder to enforce some rules